### ISSUE TITLE
denoising training?

### ISSUE THREAD

**Issue by ghost** [posenhuang/deeplearningsourceseparation#6]
Why the denoising experiment has no training code available? I am very interested in this, but would be awesome if you can provide additional code for training the denoising model. Thanks!

**Comment by posenhuang** [comment]
Thanks for your interests. The denoising setup is a bit complicated/ugly as we compare several different noisy conditions. It shouldn't be hard to follow the training examples in other tasks to try it on your data. Let me know if you have questions.

**Comment by ghost** [comment]
Thanks for reply. If I understood you correctly, are you saying that I can use the training code and generated .mat file from other experiments and use it as pre-trained model for the denoising task? If not, then I don't understand what you mean. Thanks!

**Comment by posenhuang** [comment]
No. I meant you can take your data and try to fit those in other two tasks that I prepared with training script.

**Comment by posenhuang** [comment]
FYI: I add the training code here: https://github.com/posenhuang/deeplearningsourceseparation/pull/18
